styles.css

```diff
@@ -132,6 +133,209 @@ body {
+        font-weight: 700;
+        margin: -3px 16px;
+    }
+
+    .details {
+        display: flex;
+        font-size: 13px;
+        line-height: 16px;
+        font-weight: 600;
+        margin: 0 16px;
+    }
+
+    .details ul {
+        margin: 0;
+        padding: 0;
+    }
+
+    .details li {
+        list-style-type: none;
+        display: flex;
+        align-items: center;
+        margin-top: 7px;
+    }
+
+    .details img {
+        margin: 16px 11px;
+    }
+
+    .worksText {
+        font-size: 15px;
+        line-height: 24px;
+        font-weight: 400;
+        margin: 8px 16px;
+    }
+
+    .worksTag {
+        font-size: 12px;
+        line-height: 16px;
+        font-weight: 500;
+    }
+
+    .worksTag ul {
+        margin: 2px 16px;
+        padding: 0;
+        display: flex;
+
+    }
+
+    .worksTag li {
+        list-style-type: none;
+        background-color:#EBEBFF;
+        padding: 4px 12px;
+        margin-right: 8px;
+        margin-top: 5px;
+        border-radius: 8px;
+        color: #6070FF;
+    }
+
+    .worksButton {
+        width: 124px;
+        height: 48px;
+        margin: 20px 16px;
+        font-size: 17px;
+        line-height: 24px;
+        font-weight: 500;
+        color: #396DF2;
+        border: 1px solid #6070FF;
+        border-radius: 8px;
+        background-color: #FFFFFF;
+    }
+
+    .aboutMe {
+        padding: 114px 24px;
+    }
+
+    .aboutMe h1 {
+        font-size: 40px;
+        line-height: 52px;
+        font-weight: 700;
+        margin-top: 0;
+    }
+
+    .aboutPara {
+        margin-top: -7%;
+        font-size: 15px;
+        line-height: 24px;
+    }
+
+    .aboutLink {
+        margin-top: -2.5%;
+        font-size: 16px;
+        line-height: 24px;
+        font-weight: 500;
+        color: #7F8CFF;
+    }
+
+    .aboutMe li {
+        list-style-type: none;
+        margin-left: -13.5%;
+        padding-top: 5px;
+    }
+
+    .aboutMe img {
+        height: 18px;
+        margin-right: 23px;
+    }
+
+    .aboutMeButton {
+        width: 162px;
+        height: 48px;
+        margin: 0px;
+        padding: 12px;
+        font-size: 17px;
+        line-height: 24px;
+        font-weight: 500;
+        color: #396DF2;
+        border: 1px solid #6070FF;
+        border-radius: 8px;
+        background-color: #FFFFFF;
+    }
+    
+    .aboutMeDropdown {
+        margin: 0;
+        padding: 0;
+        display: flex;
+    }
+
+    .dropdownItem {
+        font-size: 20px;
+        line-height: 24px;
+        font-weight: 500;
+    }
+
+    .dropdownItem li {
+        margin-bottom: 12px;
+        margin-left: -43px;
+        padding: 0;
+    }
+
+    .dropdownList {
+        display: flex;
+        justify-content: flex-start;
+        align-items: center;
+        width: 303px;
+        height: 67px;
+        background-color: #f7f7f9;
+        border-radius: 8px;
+        font-size: 15px;
+        line-height: 20px;
+        font-weight: 500;
+    }
```

Comment:
- [ ] Please ensure that the skill item should fill the full width of the screen as shown in the Figma design

### Your Design 
![image](https://user-images.githubusercontent.com/68030297/219882863-08303341-af7d-4d6e-b57e-f3b44442a2ca.png)

### Figma Design
![image](https://user-images.githubusercontent.com/68030297/219882879-4a565da6-8bf2-4328-b69f-a83236a0acd1.png)